uniMASK/transformer.py

```diff
@@ -84,7 +84,7 @@ def validate_training_params(tp):
     #
     #     if self.state_normalization:
     #         # Normalize (at network time, so don't have to worry about doing it at batch and eval time)
-    #         # TODO: add flag for state normalization, and add state normalization to flexiBiT too
+    #         # TODO: add flag for state normalization, and add state normalization to FB too
```

Comment:
If you ended up switching FB to UM, maybe we should do that here too

Reply:
I did not do that, just flagged that one time. We can do this separate from this PR?

Reply:
Sure yeah